migrations/1_2_0_upgrade_2_0_0.py

```diff
@@ -15,19 +15,15 @@
 from sqlalchemy_utils import (
     drop_database,
 )
+from six import string_types
 from six.moves import input
 import dataset
 
 def cast_bool(value):
     if value and value.isdigit():
         return int(value)
-    elif value and isinstance(value, six.string_types):
-        if value.lower() == 'true':
-            return True
-        elif value.lower() == 'false':
-            return False
-        else:
-            return value
+    elif value and isinstance(value, string_types):
+        return {'true': True, 'false', False}.get(value.lower(), value)
```

Comment:
Please don't alter code here. This file does not have test coverage and is sensitive for upgrading old CTFd instances.